packages/changed-elements-react/src/utils/utils.ts

```diff
@@ -69,3 +69,14 @@ export async function tryXTimes<T>(func: () => Promise<T>, attempts: number, del
   }
   throw error;
 }
+
+/** Creates map from an array of values.
+ * Expects createKey to supply a unique key per entry; otherwise will cause other entries with same key to be overwritten.
+*/
```

Comment:
```suggestion
/**
   Creates a map from an array of values.
 * Expects createKey to supply a unique key per entry; otherwise will cause other entries with same key to be overwritten.
 */
```